app/bundles/EmailBundle/Form/Type/ExampleSendType.php

```diff
@@ -39,5 +94,25 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'save_icon'  => 'fa fa-send',
             ]
         );
+
+        if (!empty($options['action'])) {
+            $builder->setAction($options['action']);
+        }
+    }
+
+    /**
+     * @param OptionsResolverInterface $resolver
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setOptional(['filter']);
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'email_example_send';
```

Comment:
Is the `getName` method needed? It's deprecated: https://github.com/symfony/symfony/pull/15079